frontend/packages/kubevirt-plugin/src/components/create-vm/forms/boot-source-form-reducer.ts

```diff
@@ -110,19 +111,19 @@ export const bootFormReducer = (
     subject: `${PersistentVolumeClaimModel.label} size`,
   });
   newState.size.validation = sizeValidation;
```

Comment:
I mentioned it already, but would be nice to reuse the validations with `validateDisk` in the future

Reply:
agreed, but not within this PR :)